prometheus/index.js

```diff
@@ -0,0 +1,80 @@
+const program = require('commander');
+const axios = require('axios');
+const querystring = require('querystring');
+const Plugin = require('nagios-plugin');
+const package = require('./package.json');
+
+let endpoint, query;
+
+const funnelPlugin = new Plugin({
+  shortName: 'prometheus'
+});
+
+program
+  .version(package.version)
+  .option('-e, --endpoint <prometheus_endpoint>', 'Prometheus endpoint (required)')
+  .option('-w, --warning <warning>', 'Warning threshold', 0)
+  .option('-c, --critical <critical>', 'Critical threshold', 0)
+  .option('-t, --time <rfc3339 | unix_timestamp>', 'Evaluation timestamp (optional)')
+  .arguments('<query>')
```

Comment:
Why not add the method arg like the original plugin?

Reply:
I just liked this syntax because it's required

Reply:
because we can know that from the warning/critical parameters